lua/wikis/commons/Infobox/Faction.lua

```diff
@@ -0,0 +1,124 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Infobox/Faction
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Flags = require('Module:Flags')
+local Json = require('Module:Json')
+local Lua = require('Module:Lua')
+local Namespace = require('Module:Namespace')
+
+local BasicInfobox = Lua.import('Module:Infobox/Basic')
+
+local Widgets = require('Module:Widget/All')
+local Cell = Widgets.Cell
+local Header = Widgets.Header
+local Title = Widgets.Title
+local Center = Widgets.Center
+local Customizable = Widgets.Customizable
+
+---@class FactionInfobox: BasicInfobox
+local FactionInfobox = Class.new(BasicInfobox)
+
+---@param frame Frame
+---@return Html
+function FactionInfobox.run(frame)
+	local faction = FactionInfobox(frame)
+	return faction:createInfobox()
+end
+
+---@return string
+function FactionInfobox:createInfobox()
+	local args = self.args
+
+	local widgets = {
+		Header{
+			name = self:nameDisplay(args),
+			subHeader = self:subHeader(args),
+			image = args.image,
+			imageDefault = args.default,
+			imageDark = args.imagedark or args.imagedarkmode,
+			imageDefaultDark = args.defaultdark or args.defaultdarkmode,
+			size = args.imagesize or 100,
```

Comment:
why default to 100?

Reply:
Default from previous AoE template version. Could change it to something else, but IMO there should be a default.

Reply:
> If you're fine with it being removed at some point in the future

Removed, or changed?

Reply:
Like, i'm fine with moving the 100 default to the AoE custom; however, i do think that there should be a default width set to avoid requesting full images instead of thumbs from the server.
We can set it to a higher value here, e.g.
```suggestion
			size = args.imagesize or 600,
```

Reply:
I'm fine with this, but we might change this everywhere soon(tm)

Reply:
Alright, then let's talk about this specific case under the circumstances then.